target/ppc/translate/vsx-impl.c.inc

```diff
@@ -2136,6 +2136,11 @@ static int pack_flags_xvi(int acc, int sat)
     return (sat << 1) | acc;
 }
 
+static int pack_masks(int pmsk, int ymsk, int xmsk)
+{
+    return (pmsk << 8) | (ymsk << 4) | (xmsk);
```

Comment:
It would be safer to mask pmsk, ymsk and xmsk to the appropriate sizes too:
```
return (pmsk & 0xFF) << 8 | (ymsk & 0xF) << 4 | (xmsk & 0xF);
```